--- transmission-1.42/configure	2008-12-24 18:53:05.000000000 +0100
+++ transmission-1.42-new/configure	2015-03-24 11:05:26.548713049 +0100
@@ -21454,8 +21454,8 @@
 
 
 if test "x$GCC" = "xyes" ; then
-    CFLAGS="-g -Wall -Wformat-security -W -Wmissing-prototypes -Wdeclaration-after-statement -O3 -funroll-loops"
-    CXXFLAGS="-g -Wall -Wformat-security -W -O3 -funroll-loops"
+    CFLAGS="$CFLAGS -g -Wall -Wformat-security -W -Wmissing-prototypes -Wdeclaration-after-statement -O3 -funroll-loops"
+    CXXFLAGS="$CFLAGS -g -Wall -Wformat-security -W -O3 -funroll-loops"
 fi
 
 { $as_echo "$as_me:$LINENO: checking for ANSI C header files" >&5
