--- cppcheck-1.52/lib/checkobsoletefunctions.cpp	2011-12-10 13:00:45.000000000 +0100
+++ cppcheck-1.52-patched/lib/checkobsoletefunctions.cpp	2015-01-21 07:42:11.173407868 +0100
@@ -47,7 +47,9 @@
         if (tok->isName() && tok->varId()==0 && (tok->next() && tok->next()->str() == "(") &&
             (!Token::Match(tok->previous(), ".|::|:|,") || Token::simpleMatch(tok->tokAt(-2), "std :: gets"))) {
             // c function declaration?
-            if ((tok->next()->link()->next() && tok->next()->link()->next()->str() == ";") && (tok->previous() && (tok->previous()->str() == "*" || tok->previous()->isName()))) {
+          const Token* prevTokPtr = tok->previous();
+          const Token prevTok = *prevTokPtr;
+          if ((tok->next()->link()->next() && tok->next()->link()->next()->str() == ";") && (prevTok.str() == "*" || tok->previous()->isName())) {
                 continue;
             }
 
