--- memcached-1.4.4/memcached.c	2009-11-27 06:45:13.000000000 +0100
+++ memcached-1.4.4-new/memcached.c	2015-03-16 16:46:19.833028230 +0100
@@ -2794,6 +2794,7 @@
         return;
     }
 
+    sleep(1);
     switch(add_delta(c, it, incr, delta, temp)) {
     case OK:
         out_string(c, temp);
@@ -2862,7 +2863,8 @@
         memcpy(ITEM_data(new_it), buf, res);
         memcpy(ITEM_data(new_it) + res, "\r\n", 2);
         item_replace(it, new_it);
-        do_item_remove(new_it);       /* release our reference */
+        free(new_it);
+        //do_item_remove(new_it);       /* release our reference */
     } else { /* replace in-place */
         /* When changing the value without replacing the item, we
            need to update the CAS on the existing item. */
